src/Mutator/Arithmetic/Assignment.php

```diff
@@ -68,8 +68,6 @@ public static function getDefinition(): ?Definition
     }
 
     /**
-     * @psalm-mutation-free
```

Comment:
all `@psalm-mutation-fee` generates
```
ERROR: ImpureMethodCall - src/Mutator/Arithmetic/MinusEqual.php:76:75 - Cannot call an possibly-mutating method PhpParser\Node::getAttributes from a mutation-free context (see https://psalm.dev/203)
        yield new Node\Expr\AssignOp\Plus($node->var, $node->expr, $node->getAttributes());
```

Reply:
Let's recall we've added these in #1382 on purpose because we have very annoying and hard to debug bugs.